packages/flutter_tools/lib/executable.dart

```diff
@@ -66,6 +67,7 @@ Future<Null> main(List<String> args) async {
     ..addCommand(new RunCommand())
     ..addCommand(new RunMojoCommand(hidden: !verboseHelp))
     ..addCommand(new ScreenshotCommand())
+    ..addCommand(new SkiaCommand())
```

Comment:
Is this something we expect most developers to use? It's not immediately clear to me what a command called `skia` would do. Most of our other commands are verbs. Perhaps we could orient this towards what the developer will be using it for? `diagnose-graphics`, `drawing-debug`, `render`?